jwt/proxy_handlers.go

```diff
@@ -223,11 +223,15 @@ func NewAuthenticationHandler(cfg config.VerifierConfig) (*StoppableProxyHandler
 			}
 
 			cookie := http.Cookie{Name: "access_token", Value: token, HttpOnly: true, Path: cookiePath}
+			var cookieString string
 			if redirectUrl.Scheme == "https" {
 				cookie.Secure = true
+				cookieString = cookie.String() + "; SameSite=None"
+			} else {
+			        cookieString = cookie.String()
```

Comment:
Does it make sense to always set SameSite and not rely on browser default?
I can wonder when someday a default value for `SameSite` will be `Strict` and then we'll need to fix it as well.

So, it's better to learn what SameSite is and configure the value we really need.
P.S. I did not learn much about SameSite, so my comment may be not valid at all.

Reply:
I hope we will be able to use just `cookie.SameSite = http.SameSiteNoneMode` when able to use newer Golang version

Reply:
Well, we won't since Google Chrome does not allow it ) https://www.chromestatus.com/feature/5633521622188032

So, we always will have this if, but it won't be possible to use Che in multi-cluster mode without https

Reply:
It do allow, but only together with `Secure`. So my first commit is tested and works fine: https://github.com/eclipse/che-jwtproxy/pull/23/commits/4f710d91f675117dc478adf1fb422f3b12d9b014

Reply:
Sure, I understand. Probably I misunderstood your statement

> I hope we will be able to use just cookie.SameSite = http.SameSiteNoneMode when able to use newer Golang version

What do you mean? Do you mean that we'll set none always?

Reply:
I mean we will be able to use language constant instead of hardcode, so even if it occasionally change, we [maybe] will not be forced to support backward compartibility :)

Reply:
> I mean we will be able to use language constant instead of hardcode, so even if it occasionally change, we [maybe] will not be forced to support backward compartibility :)

And I meant 

```go
			var cookieString string
			if redirectUrl.Scheme == "https" {
				cookie.Secure = true
				cookieString = cookie.String() + "; SameSite=None"
			} else {
				cookieString = cookie.String() + "; SameSite=Lax"
                        }
```

Reply:
Ok. that makes sense.